src/Kiosk.Repositories/NewsRepository.cs

```diff
@@ -20,10 +21,17 @@ public NewsRepository(IMongoDatabase mongoDatabase)
         => await _news.Find(news => news._id == id)
             .FirstOrDefaultAsync(cancellationToken);
 
-    public async Task<IEnumerable<News>?> GetManyNews(Source? source, CancellationToken cancellationToken)
+    public async Task<(IEnumerable<News>?, Pagination Pagination)> GetManyNews(Source? source, Pagination pagination, CancellationToken cancellationToken)
     {
         Expression<Func<News, bool>> filter = news => source == null || news.Source.ToString() == source.ToString();
```

Comment:
It shouldn't be `var filter = Builders<News>.Filter.Eq(r => r.source == null || r.source.ToString() == source.ToString(););`

Reply:
it doesnt work that way